moped-toolbox/produce_current_pk_sequences/script/gen_pk_sequences.pl

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/perl
```

Comment:
This started as a throw-away script. I can port it to python if needed or removed from this PR.